src/addremove.js

```diff
@@ -0,0 +1,83 @@
+export const toDoList = JSON.parse(localStorage.getItem('toDoList')) || [];
+
+let index = toDoList.length;
+
+export const generateElement = (index, completed, description) => {
+  const ulLists = document.querySelector('.ul-lists');
+  const list = document.createElement('li');
+  list.setAttribute('class', 'list');
+  list.setAttribute('id', `${index}`);
+  ulLists.appendChild(list);
+  const span = document.createElement('span');
+  span.setAttribute('class', 'list-check');
+  list.appendChild(span);
+
+  const checkbox = document.createElement('input');
+  checkbox.setAttribute('type', 'checkbox');
+  checkbox.className = 'checkbox';
+  checkbox.checked = completed;
+  span.appendChild(checkbox);
+
+  const paragraph = document.createElement('input');
+  paragraph.value = description;
+  paragraph.setAttribute('class', 'para-input');
+  span.appendChild(paragraph);
+
+  const ellipsis = document.createElement('i');
+  ellipsis.setAttribute('class', 'fa fa-ellipsis-v');
+  list.appendChild(ellipsis);
+};
+
+export const addItem = (description, completed) => {
+  const item = { description, index, completed };
+  if (description !== '') {
+    index += 1;
+    item.index = index;
+    toDoList.push(item);
+    localStorage.setItem('toDoList', JSON.stringify(toDoList));
+    generateElement(index, item.completed, item.description);
+  }
+};
+
+export const removeTask = (lists) => {
+  lists.forEach((elt) => {
+    elt.addEventListener('click', () => {
+      lists.forEach((elt2) => {
+        elt2.style.background = '';
+        elt2.lastElementChild.setAttribute('class', 'fa fa-ellipsis-v');
+        elt2.querySelector('span').querySelector('.para-input').style.background = '';
+      });
+      elt.style.background = '#fffeca';
+      elt.querySelector('span').querySelector('.para-input').style.background = '#fffeca';
+      elt.lastElementChild.classList = 'fa fa-trash';
+    });
+  });
+  for (let i = 0; i < lists.length; i += 1) {
```

Comment:
I think you can still use foreach loop here. Please check this [link](https://medium.com/infancyit/kiss-keep-it-simple-smart-javascript-habits-b019ce14b5ec) number 3 example.